scalability_and_performance/enabling-workload-partitioning.adoc

```diff
@@ -6,57 +6,34 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In resource-constrained environments, you can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+Workload partitioning segregates particular workloads into distinct CPU sets. The primary objective is ensure that the platform pods will not exceed a specified CPU usage limit. You can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs. This ensures that the remaining CPUs in their deployment remain untouched and available exclusively for their own workloads. The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
 
-The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
-With workload partitioning, you annotate the set of cluster management pods and a set of typical add-on Operators for inclusion in the cluster management workload partition.
-These pods operate normally within the minimum size CPU configuration.
-Additional Operators or workloads outside of the set of minimum cluster management pods require additional CPUs to be added to the workload partition.
+In the context of enabling workload partitioning and managing CPU resources effectively, nodes within the cluster must identify themselves as supporting a new resource type called `management.workload.openshift.io/cores` and accurately report their CPU capacity. Workload partitioning can be enabled during cluster installation only by adding the additional field: `cpuPartitioningMode` to the `install-config.yaml` file.
 
-Workload partitioning isolates user workloads from platform workloads using standard Kubernetes scheduling capabilities.
+When workload partitioning is enabled, the `management.workload.openshift.io/cores` resource allows the scheduler to correctly assign pods based on the cpushares capacity of the host, not just the default cpuset. This ensures more precise allocation of resources for workload partitioning scenarios.
 
-The following changes are required for workload partitioning:
+Workload partitioning respects CPU requests and limits defined on pods. From 4.16 accurate CPU usage limitations are set for platform pods utilizing CPU partitioning. Resource requests, rather than limits, are the primary value in setting accurate CPU usage limitations. 
```

Comment:
Maybe rework this paragraph a bit,

Something like this, maybe?


> Workload partitioning respects CPU requests and limits defined in the pod spec. As of 4.16 accurate CPU usage limits are set for platform pods utilizing CPU partitioning. Since workload partitioning uses the custom resource type of `management.workload.openshift.io/cores`,  the values for `requests` and `limits` will equal the same due to a requirement by Kubernetes for extended resources. However, the annotations modified by workload partitioning will correctly reflect the desired limits.